pkg/fdbadminclient/mock/admin_client_mock.go

```diff
@@ -803,3 +823,54 @@ func (client *AdminClient) ResetMaintenanceMode() error {
 func (client *AdminClient) MockUptimeSecondsForMaintenanceZone(seconds float64) {
 	client.uptimeSecondsForMaintenanceZone = seconds
 }
+
+// SetLimitingDurabilityLag sets/mocks the limiting durability lag of any storage server in the cluster.
+func (client *AdminClient) SetLimitingDurabilityLag(lagInfo *fdbv1beta2.FoundationDBStatusLagInfo) {
+	adminClientMutex.Lock()
+	defer adminClientMutex.Unlock()
+
+	client.LagInfo["limitingDurabilityLag"] = *lagInfo
+}
+
+// GetLimitingDurabilityLag returns the limiting durability lag of any storage server in the cluster.
+func (client *AdminClient) GetLimitingDurabilityLag() (fdbv1beta2.FoundationDBStatusLagInfo, bool) {
+	adminClientMutex.Lock()
+	defer adminClientMutex.Unlock()
+
+	lagInfo, ok := client.LagInfo["limitingDurabilityLag"]
+	return lagInfo, ok
```

Comment:
```suggestion
	return client.LagInfo["limitingDurabilityLag"]
```

Reply:
Actually I tried that yesterday - it was complaining about "not enough return values". I still get the same error.

Reply:
Right, I forgot that in this case the internals will not return the boolean value.